helm/kong-app/CHANGELOG.md

```diff
@@ -2,9 +2,39 @@
 
 ## Unreleased
 
+Nothing yet.
+
+## 2.8.1
```

Comment:
Why are there two entries for `2.8.1`? Can we merge those?

Reply:
This is an error in the upstream repository. Do you think it should be fixed in ours?

Reply:
Ah ok, no need then.